### DISCUSSION TITLE
Error when running Deploy-SignedWDACConfig

### DISCUSSION THREAD

**Discussion by jeffreyflynt** [HotCakeX/Harden-Windows-Security#279] [Q&A] [answered]
When I run: 

```
Deploy-SignedWDACConfig -CertPath "C:\Program Files\WDACConfig\Code Signing Certificate.cer" -PolicyPaths "C:\Program Files\WDACConfig\DefaultWindows.xml" -CertCN "Code Signing Certificate" -Deploy
```

I get this error:
`Get-SignTool: Cannot convert the "System.Object[]" value of type "System.Object[]" to type "System.IO.FileInfo"`

Any ideas?

Thanks

**Comment by HotCakeX** [comment, marked as answer]
Hi @jeffreyflynt 

Can you please run this command

```powershell
Get-CommonWDACConfig -SignToolPath
```

and see what is displayed? I ran the same command you gave me multiple times and couldn't reproduce the error. So i'm thinking the user configuration json file has somehow stored a wrong value for SignTool.exe path.

To clear the SignTool path in the user config, you can run the following command

```powershell
Remove-CommonWDACConfig -SignToolPath
```

To completely clear the user configs, you can run the following command

```powershell
Remove-CommonWDACConfig
```

To open the user config file in the default text editor, you can run the following command

```powershell
Get-CommonWDACConfig -Open
```

**Reply by jeffreyflynt** [reply]
Thanks, that put me on the right path. There was no signtool path set.

**Reply by HotCakeX** [reply]
Yw, i wonder why it still tried to use the user config signtool path if it wasn't set there, it should've offered to download it for you from nuget servers..
